restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/Call.java

```diff
@@ -1748,17 +1748,8 @@ public void execute(Object message) throws Exception {
             }
 
             msController.tell(new CloseMediaSession(), source);
-            if (fail) {
-                if (logger.isDebugEnabled()) {
-                    logger.debug("At Call Stopping state, moving to Failed state");
-                }
-                fsm.transition(message, failed);
-            } else {
-                if (logger.isDebugEnabled()) {
-                    logger.debug("At Call Stopping state, moving to Completed state");
-                }
-                fsm.transition(message, completed);
-            }
+            //Don't wait for ever for the CallController response
+            context().setReceiveTimeout(Duration.create(2000, TimeUnit.SECONDS));
```

Comment:
this kind of timeouts should come from configuration, so we can tweak then (or customers) depending on envirnoment... where is 2 secs coming from (for me is regular call timeout, just wondering if this is iline with some other system in the cloud or something)=

Reply:
@jaimecasero this is a safety net for Conference in a cluster configuration where notification from media server might never arrive. It doesn't need to be on configuration since this is an internal, very specific timer to be able to cleanup call no matter what error condition might occur. No particular reason for for the 2 sec duration, I can reduce it if you think its too much

Reply:
2 seconds seems ok